linuxNew/README.md

```diff
@@ -1,10 +1,78 @@
-# Linux Packages of Eclipse Adoptium 
+# Linux Packages of Eclipse Adoptium
 
-## RPMs for Fedora/Red Hat
+We package for Debian, Red Hat, SUSE (e.g. DEB and RPM based) Linux distributions.
 
-### Other Topics
+The current implementation to build the packages involves using Gradle to call a small Java program.
+That Java program spinning up a Docker container, installing the base O/S and its packaging tools,
+and then looping over configuration to create the various packages and sign them as appropriate
+with the (Eclipse Foundation as a default) signing service.
```

Comment:
In line with the above, sticking with the existing tense ("signing them as appropriate") might be slightly preferable.

Reply:
Fixed locally thanks